addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/emr-cluster.cfn.yml

```diff
@@ -215,6 +215,7 @@ Resources:
                 Action: 's3:GetObject'
                 Resource:
                   - !Sub 'arn:aws:s3:::${AWS::Region}.elasticmapreduce/bootstrap-actions/run-if'
+                  - 'arn:aws:s3:::elasticmapreduce/bootstrap-actions/log4j/patch-log4j-emr-5.27.1-v1.sh'
```

Comment:
Add a comment here explain the owner of this bucket and script? 

Also, on line 287, the bucket name used is region specific, not sure if region specific bucket name should be used here as well?

Reply:
This is a standard path for bootstrap-actions script owned by EMR team (`s3://elasticmapreduce/bootstrap-actions`)

I tried using the region specific endpoint and arn but it didn't work. I got the bootstrap script from EMR documentation: https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-log4j-vulnerability.html